main.tex

```diff
@@ -1012,12 +1019,12 @@ \subsubsection*{Provenance of individual \wnt{} simulation models}
 This could be achieved, for example, by publishing a COMBINE archive~\cite{Bergmann2014}, which is a \enquote{single file that aggregates all data files and information necessary to reproduce a simulation study in computational biology}~\cite{Scharm2016b}.
 However, the publications we have analyzed date back up to 17 years, so most data has not been published in a FAIR way.
 
-When looking at \WD{}, six out of 19 publications recorded own wet-lab data \cite{Lee2003, Sick2006, Kim2007, Kogan2012, Haack2015, Staehlke2020}.
+When looking at \WD{}, six out of 19 publications recorded their own wet-lab data \cite{Lee2003, Sick2006, Kim2007, Kogan2012, Haack2015, Staehlke2020}.
 All other simulation studies either used wet-lab data from other publications or did not explicitly refer to wet-lab data at all \cite{Kruger2004, Cho2006, Rodriguez2007, vanLeeuwen2007, vanLeeuwen2009, Mirams2010, Padala2017}.
-The latter could only be done because the authors relied on other \SMs{} and the respective parameter and initial values.
-Interestingly, the wet-lab data obtained either by the 19 \wnt{} signaling studies or by other studies which where used in the 19 studies stemmed from different organisms and cell lines.
+The latter could only be done because the authors relied on other \SMs{} and their respective parameters and initial values.
```

Comment:
parameter and initial values = parameter values and initial values. Therefore "parameter" instead of "parameters"?!

Reply:
oh yup!! you're right